openfoam-interface/internal-interface/wrapper/simpleFoam.H

```diff
@@ -0,0 +1,155 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
@santiagomarquezd , I don't see any additonal commits that would have changed this.  Have you not pushed this to GitHub yet?